assets/css/main.eidolon.css

```diff
@@ -143,33 +156,38 @@ body {
   margin-bottom: 40px !important;
   color: #00addd;
   font-size: 17px;
-  text-shadow: 1px 1px 1px rgba(13, 13, 13, 0.2);
+  text-shadow: 1px 1px 1px rgba(13, 13, 13, .2);
 }
 
 .panel {
-  background-color: rgba(12, 12, 12, 0.3);
+  background-color: rgba(12, 12, 12, .3);
 }
 
 .panel-title {
 	color: #13bdec;
 }
 
 .panel-primary {
-  border-color: rgba(12, 12, 12, 0.7);
+  border-color: rgba(12, 12, 12, .7);
 }
 
 .panel-primary>.panel-heading {
-  background-color: rgba(2, 48, 59, 0.98);
-  border-color: rgba(2, 48, 59, 0.98);
+  background-color: rgba(2, 48, 59, .7);
+  border-color: rgba(2, 48, 59, .9);
 }
 
 .table-striped>tbody>tr:nth-of-type(odd) {
-  background-color: rgba(12, 12, 12, 0.5);
+  background-color: rgba(12, 12, 12, .5);
 }
 
 .table-striped>tbody>tr:nth-of-type(odd):hover,
 .table-striped>tbody>tr:nth-of-type(even):hover {
-  background-color: rgba(0, 120, 153, 0.2);
+  background-color: rgba(0, 120, 153, .2);
+}
+
+.panel-heading a,
+.panel-heading a:hover {
+  text-decoration: none;
```

Comment:
might just want to add this to the common.css since it'd be better for all of them to do this